templates/zerver/app/markdown_help.html

```diff
@@ -92,13 +92,11 @@
                         <td class="rendered_markdown">Some inline <code>code</code></td>
                     </tr>
                     <tr>
-                    <td class="preserve_spaces">```
+                    <td class="preserve_spaces apply_markdown">```
 def zulip():
     print "Zulip"
 ```</td>
                     <td class="rendered_markdown">
-                        <div class="codehilite"><pre>def zulip():
-    print "Zulip"</pre></div>
```

Comment:
We should be able to replace all the `rendered_markdown` blocks` in this file, not just a couple, right?

Reply:
Do we want to replace all the `rendered_markdown` blocks, even the `Italic`, `bold`, `strikethrough` and many others?

Reply:
Yes, that's clearly a cleaner implementation if we can just render everything using the markdown processor rather than manual formatting of its output.

Reply:
I think the blocks having some written instructions like `<td>*italic* (or <kbd>Ctrl + I</kbd>)</td>`, they can not be rendered from the parser. As this will give different output from what we want to show.